### ISSUE TITLE
Obsolete android command used instead of avdmanager

### ISSUE THREAD

**Issue by koral--** [bitrise-steplib/steps-start-android-emulator#29]
Emulators are tried to be created with `android` command which is no longer usable in current version of SDK tools. `tools/bin/avdmanager` is a successor.
Sample log: https://www.bitrise.io/build/35f264d2fcc5b7ed

**Comment by godrei** [comment]
Hi @koral-- ,
thanks for the info!
I scheduled to fix this!

**Comment by viktorbenei** [comment]
Related log:

```
+------------------------------------------------------------------------------+
| (4) create-android-emulator@1.0.0                                            |
+------------------------------------------------------------------------------+
| id: create-android-emulator                                                  |
| version: 1.0.0                                                               |
| collection: https://github.com/bitrise-io/bitrise-steplib.git                |
| toolkit: go                                                                  |
| time: 2017-03-07T00:08:19Z                                                   |
+------------------------------------------------------------------------------+
|                                                                              |
[34mINFO[0m[00:08:19] Start installing (golang) with apt-get       
[34mINFO[0m[00:08:19]  * [32;1m[OK][0m Step dependency (go) installed, available. 

[34;1mConfigs:[0m
- Name: test
- Platform: android-25
- Abi: armeabi-v7a
- Options: 
- CustomHardwareProfileContent:
- AndroidHome: /opt/android-sdk-linux


[34;1mCheck if platform installed[0m
[32;1minstalled: true[0m

[34;1mCheck if system image installed[0m
checking path: /opt/android-sdk-linux/system-images/android-25/armeabi-v7a
checking path: /opt/android-sdk-linux/system-images/android-25/default/armeabi-v7a
[32;1minstalled: false[0m

[34;1mInstalling: sys-img-armeabi-v7a-android-25[0m

[32;1m$ android "update" "sdk" "--no-ui" "--all" "--filter" "sys-img-armeabi-v7a-android-25"[0m

The android command is no longer available.
For manual SDK and AVD management, please use Android Studio.
For command-line tools, use tools/bin/sdkmanager and tools/bin/avdmanager
[31;1mFailed to install system image, error: exit status 1[0m
|                                                                              |
+---+---------------------------------------------------------------+----------+
| [31;1mx[0m | [31;1mcreate-android-emulator@1.0.0 (exit code: 1)[0m                  | 1.67 sec |
+---+---------------------------------------------------------------+----------+
| Issue tracker: ...b.com/bitrise-steplib/steps-create-android-emulator/issues |
| Source: https://github.com/bitrise-steplib/steps-create-android-emulator     |
+---+---------------------------------------------------------------+----------+
```

It seems it affects android-25 and probably newer, does not affect android-24

**Comment by godrei** [comment]
Hi @koral-- ,
we released a new version ([v1.2.0](https://github.com/bitrise-steplib/steps-start-android-emulator/releases/tag/1.2.0)), which uses the correct sdk tools based on the sdk versions.

Thanks for the report!
